lib/utils.js

```diff
@@ -141,13 +141,11 @@ api.fetchInSecurityContext = async ({
   url, isRoot, documentLoader, expansionMap
 }) => {
   if(url && typeof url === 'object' &&
+  // skip compaction when using specific contexts, this is a temporary
+  // measure until a zcap context is created
```

Comment:
Is this comment still correct?  Seems odd now since the code is using a zcap context.